apps/src/templates/PaneHeader.jsx

```diff
@@ -83,42 +82,39 @@ var styles = {
 };
 
 /**
- * A purple pane header that can have be focused (purple), unfocused (light purple)
- * or read only (charcoal).
+ * A purple pane header that can have be focused (purple) or unfocused (light purple).
  */
-const PaneHeader = Radium(React.createClass({
-  propTypes: {
+class PaneHeader extends React.Component {
+  static propTypes = {
     hasFocus: PropTypes.bool.isRequired,
-    readOnly: PropTypes.bool,
     style: PropTypes.object
-  },
+  };
 
-  render: function () {
-    var {hasFocus, readOnly, style, ...props} = this.props;
+  render() {
+    let {hasFocus, style, ...props} = this.props;
 
     // TODO: AnimationTab should likely use components from PaneHeader, at
     // which point purpleHeader style should move in here.
     style = [
       style,
       commonStyles.purpleHeader,
-      !hasFocus && commonStyles.purpleHeaderUnfocused,
-      readOnly && commonStyles.purpleHeaderReadOnly,
+      !hasFocus && commonStyles.purpleHeaderUnfocused
     ];
```

Comment:
Optional: Instead of depending on Radium to compose these styles we can do it with spread syntax:

```jsx
const {hasFocus, style, ...props} = this.props;
const composedStyle = {
  ...style,
  ...commonStyles.purpleHeader,
  ...(!hasFocus && commonStyles.purpleHeaderUnfocused),
};
return <div {...props} style={composedStyle}/>;
```

Reply:
Great call! I ended up updating this in a bunch of places in this file - thanks!